dbcon/execplan/constantfilter.cpp

```diff
@@ -272,8 +272,19 @@ void ConstantFilter::setDerivedTable()
         fDerivedTable = "";
         return;
     }
+    for (unsigned i = 0; i < fFilterList.size(); i++)
```

Comment:
Are you sure we need this here?

Reply:
Seems this commit was accidentally pushed up. this is a part of MCOL-2096. Removing